device_backends/LogicalNameMapping/include/LNMBackendVariableAccessor.h

```diff
@@ -150,6 +163,18 @@ namespace ChimeraTK {
       });
     }
 
+    // make sure FormulaHelpers for MathPlugin instances involving this variable as push-parameter are created
+    auto& lnmVariable = _dev->_variables[_info.name];
+    for(auto* mp : lnmVariable.usingFormulas) {
+      if(mp->_hasPushParameter) {
+        // following check eliminates recursion, getFormulaHelper also creates Variable accessors
```

Comment:
There is not many places where _creatingFormulaHelper is accessed. Also, if you do it wronly and get infinite recursion you will notice immediately, since it causes a crash.